test/lib/y2firewall/clients/auto_test.rb

```diff
@@ -399,4 +399,10 @@
       end
     end
   end
+
+  describe "#packages" do
```

Comment:
It was not related to the fix, but decreasing the test coverage is not nice :laughing: . So, this was added just for increasing the test coverage slightly.